.github/workflows/nix-jupyterlab.yml

```diff
@@ -0,0 +1,27 @@
+name: nix-jupyterlab
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v6
+    - name: Setup repo
+      run: |
+        git submodule init && git submodule update
+    - uses: cachix/cachix-action@v2
+      with:
+        name: hasktorch
+        signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+        file: experimental/jupyterlab/default.nix
+        nixBuildArgs: --option sandbox false 
```

Comment:
This pr pushes the caches of both jupyter and hasktorch to reduce compilation time.

Reply:
https://github.com/hasktorch/hasktorch/pull/257/checks?check_run_id=476444227#step:5:15245
Why sandbox=false doesn't work?

Reply:
Oh, nixBuildArgs is cachix/cachix-action@v4's function.